source/DG.Core/Extensions/SerializerExtensions.cs

```diff
@@ -14,5 +16,49 @@ public static T FromJson<T>(this string jsonData)
         {
             return JsonSerializer.Deserialize<T>(jsonData);
         }
+
+        public static JToken GetJTokenByPath(this JObject jObject, string pathToJObject)
+        {
+            JToken jToken = null;
+            var pathToSectionDelimiter = ':';
+            var pathElements = pathToJObject.Split(pathToSectionDelimiter);
+            for (int i = 0; i < pathElements.Length; i++)
+            {
+                if (i == 0)
+                {
+                    jToken = jObject[pathElements[i]];
+                    if (jToken == null)
+                    {
+                        throw new ArgumentException($"The path {pathElements[i]} is invalid");
+                    }
+                }
+                else
```

Comment:
We can pull this `if(i == 0)` out by doing first iteration outside of cycle:
`jToken = jObject[pathElements[0]];`
`                    if (jToken == null)`
`                    {`
`                        throw new ArgumentException($"The path {pathElements[i]} is invalid");`
`                    }`
`for (int i = 1; i < pathElements.Length; i++)`
 `           {`

Reply:
Yes, it really simplifies, thank you!